src/ray/gcs/client_def.h

```diff
@@ -0,0 +1,46 @@
+#ifndef RAY_GCS_CLIENT_DEF_H
+#define RAY_GCS_CLIENT_DEF_H
+
+#include <boost/optional/optional.hpp>
+#include <string>
+#include <vector>
+#include "ray/gcs/tables.h"
+
+namespace ray {
+
+namespace gcs {
+
+class ClientOption {
+ public:
+  // GCS server list: <ip, port>
+  std::vector<std::pair<std::string, int>> server_list_;
+
+  // Password of GCS server.
+  std::string password_;
+  // GCS command type. If CommandType::kChain, chain-replicated versions of the tables
+  // might be used, if available.
+  CommandType command_type_ = CommandType::kChain;
```

Comment:
should defaults to `kRegular`?

Reply:
> should defaults to `kRegular`?

see the now code

Reply:
then we should leave it empty here?

Reply:
actually init with any invalid value is better than uninit, other wise unknown error might occur and it's hard to track.